docs/release-announcement.md

```diff
@@ -1,11 +1,17 @@
 # Release announcement
 
-## Azure Industrial IoT Platform 2.8 Release
+## Azure Industrial IoT Platform Release 2.8.1
+We are pleased to announce the release of version 2.8.1 of our Industrial IoT Platform components as the first patch update of the 2.8 Long-Term Support (LTS) release. This release contains important security updates, bugfixes and performance optimizations.
 
-We are pleased to announce the release of version 2.8.0 of our Industrial IoT Platform as well as the declaration of Long-Term Support (LTS) for this version.
+> IMPORTANT
+> Please note that OPC Publisher 2.8.1 is not backwards compatible with version 2.5.x. This will be addressed in the next patch release.
+
+## Azure Industrial IoT Platform Release 2.8
+
+We are pleased to announce the release of version 2.8 of our Industrial IoT Platform as well as the declaration of Long-Term Support (LTS) for this version.
 While we continue to develop and release updates to our ongoing projects on GitHub, we now also offer a branch that will only get critical bug fixes and security updates starting in July 2021. Customers can rely upon a longer-term support lifecycle for these LTS builds, providing stability and assurance for the planning on longer time horizons our customers require. The LTS branch offers customers a guarantee that they will benefit from any necessary security or critical bug fixes with minimal impact to their deployments and module interactions. At the same time, customers can access the latest updates in the main branch to keep pace with the latest developments and fastest cycle time for product updates.
 
 > IMPORTANT
-> We suggest to update from the version 2.6 or later to ensure secure operations of your deployment. 2.8.0 is not backwards compatible with version 2.5.
+> We suggest to update from the version 2.6 or later to ensure secure operations of your deployment. 2.8 is not backwards compatible with version 2.5.
```

Comment:
Same comment as above, let's keep `2.8.0` versioning instead of just `2.8` as this announcement is specifically about `2.8.0`.

Reply:
Same as above.

Reply:
Reverted.